Src/HALALMock/Services/SharedMemory/SharedMemory.cpp

```diff
@@ -1,8 +1,34 @@
 
 #include "HALALMock/Services/SharedMemory/SharedMemory.hpp"
-
+//includes to create the shared Memory in Posix
+#include <fcntl.h>
+#include <sys/shm.h>
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <unistd.h>
 void SharedMemory::start() {
-    // TODO: create / open shared memory
+    //Create GPIO_Memory
+	const char* 
```

Comment:
??

Reply:
what gonzalo said "??", this line seems to be a leftover from something you were doing